examples/data_source_lacework_metrics/main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+provider "lacework" {}
+
+data "lacework_metrics" "test" {
```

Comment:
nit: Should we make the data source name singular? The name in plural makes it sound we are collecting multiple metrics, but in reality is only one.